src/purple/handlers/traits.rs

```diff
@@ -51,6 +52,10 @@ pub trait GetChatNameHandler {
     fn get_chat_name(data: Option<&mut StrHashTable>) -> Option<String>;
 }
 
+pub trait HandleCommandHandler {
+    fn handle_command(&mut self, cmd: &str, args: Vec<&str>) -> purple_sys::PurpleCmdRet;
```

Comment:
While we give us trouble to convert `String` to `&str`, might as well use a slice here! `args: [&str]`. This way all parameters a reference and no owned value (Such as `Vec`) are passed.

Reply:
Done!